nano/node/election.cpp

```diff
@@ -281,7 +281,10 @@ bool nano::election::transition_time (nano::confirmation_solicitor & solicitor_a
 		result = true;
 		state_change (state_m.load (), nano::election::state_t::expired_unconfirmed);
 		status.type = nano::election_status_type::stopped;
-		log_votes (tally ());
+		if (node.config.logging.election_expiration_tally_logging ())
+		{
+			log_votes (tally (), "Election expired");
```

Comment:
This should probably be "Election expired: " otherwise it smashes in to "Vote tally..."